assets/css/edd-admin.css

```diff
@@ -3706,3 +3685,78 @@ div:not(#edd-item-tab-wrapper) + #edd-item-card-wrapper .item-section { /** [1]
 	font-style: italic;
 	width: 150px;
 }
+
+/**
+ * Forms
+ * ---------------------------------------------------------- */
+
+/**
+ * Form Group
+ *
+ * <div class="edd-form-group">
```

Comment:
Huh. I think I really like having the HTML in here as a reference. I never thought to do that!

Reply:
I think that was @spencerfinnell s doing.